instagram-app/src/components/PostContainer/Post.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+
+import "./PostContainer.css";
+
+import PostTop from "./PostTop";
+import PostContent from "./PostContent";
+import CommentSection from "../CommentSection/CommentSection";
+
+const Post = props => {
+  return (
+    <div className="post-border">
+      <PostTop 
+      username={props.postData.username}
+      thumbnail={props.postData.thumbnailUrl}
+      />
+     <PostContent 
+     image={props.postData.imageUrl}
+     likes={props.postData.likes}
+     />
+     {props.postData.comments.map(comment => (
+      <CommentSection 
+      comments={comment}
+      key={comment.id}
```

Comment:
No need to have all these components. While breaking down into multiple components is good, there's no need to over do it. All of this can fit into one file.